starters/angular-apollo-tailwind/src/app/home/home.component.html

```diff
@@ -5,9 +5,7 @@ <h1 class="text-3xl text-white py-6">
         </h1>
     </div>
     <div class="mt-4 flex flex-col">
-        <a href="/counter" class="underline hover:text-blue-700">
-            See Counter Example Component
-        </a>
+        <a href="/counter">See Counter Example Component</a>
         <a href="/api-example" class="underline hover:text-blue-700">
```

Comment:
You should remove these classes too per the above change.

Reply:
Corrected.